### ISSUE TITLE
Is V2 ready for production ? 

### ISSUE THREAD

**Issue by almogdesign** [gadicc/meteor-messageformat#150]
Wanted to know which version I should be using in a new project?

**Comment by DSpeichert** [comment]
Although not officially stable, I believe v2 is close to being finished. I've heard of many people using it in production, including myself. I'm convinced any new bugs that might occur are promptly resolved before a new version is released to atmosphere. IMO, features in v2 are worth the risk.

**Comment by Maxhodges** [comment]
how to compile the translations in v2?

**Comment by almogdesign** [comment]
@Maxhodges what do you mean by compile the translations this happens automatically by the extra package

**Comment by Maxhodges** [comment]
OK I see. working now. We had to put in some special helpers to make it work reactively with autoform

Btw something weird happened in the editor. Should I post this as a new issue?
http://screencast.com/t/1zZbwd8PL2

You can see the labels I click on are not the labels it opens in the editor. It's like the ordinal index numbers for the rows got messed up or something. A refresh fixed it, but  there must be for bug for this to happen in the first place.

**Comment by almogdesign** [comment]
Yea that looks like a bug, open a ticket if it happens again. Bye the how is autoform working for you? I had to rollout my own html form

**Comment by Maxhodges** [comment]
too soon to say for sure, but we are having some success with this approach

in an autoform quickfield section of a template

```
 {{> afQuickField name='productName' value=preProductName label=mfProductName}} 
```

in the template helpers

```
 mfProductName: function(){
    return mf('lineItem.productName');
  },
 mfProductName: function(){
    return mf('lineItem.productName');
  },
```

**Comment by almogdesign** [comment]
Thanks I run into too many issues when trying to do steps wizard, some advanced select2 options and so forth.

**Comment by Maxhodges** [comment]
i see. Yeah we might have to do those manually but can still leverage quickfield with these helpers!

**Comment by Maxhodges** [comment]
added the helper technique to the wiki. Maybe you can share your approaches there too
https://github.com/gadicc/meteor-messageformat/wiki/messageFormat-with-autoform

**Comment by gadicc** [comment]
hi all sorry for the lack of response, i'm away on holiday at the moment... i'm back next week and will respond properly to all the open issues.  appreciate your patience.

**Comment by Maxhodges** [comment]
@gadicc Hope you have a good vacation! Looking forwarding to having you back. I found a few issues with V2, but I also have a few questions about best practices regarding nested html and template helpers inside mf strings, and some confusion about how duplicates are handled. Cheers!

**Comment by Maxhodges** [comment]
@gadicc when are you back?

**Comment by 1u** [comment]
What is missing in order to release 2.0 ?
Is there feature-parity between v0 and v2?

DSpeichert commented on 11 Aug 2015 (in this Issue)
> Although not officially stable, I believe v2 is close to being finished. I've heard of many people using it in production, including myself.

gadicc commented on 22 Mar 2015 (in 101 and in 102 on new ui-features)
v0 is feature frozen though and I can't start working on any of this until v2 achieves feature parity with v0, and there is no ETA for that at the moment

**Comment by gadicc** [comment]
Hey, sorry for late reply.  Yes, not only did we achieve feature parity, there are already a lot of new features in the v2 branch now!  Unfortunately I still can't give an ETA, but yes, a lot of people are using this in production already, and I'm happy to report that we're off to a [great start](https://github.com/gadicc/meteor-messageformat/commits/v2) this year (big thanks to all our contributors and especially @DSpeichert, @MartinFournier, @1u).

Here is the list of issues we want to close before a v2 release (updated now):
https://github.com/gadicc/meteor-messageformat/milestones/v2
